### PR TITLE
Uplinks now play a unique 'kaching' sound when purchasing any items.

### PR DESCRIPTION
**Pull request by hyperjll**
## About The Pull Request

Syndicate (and others) Uplinks now make a 'kaching' sound whenever the user purchases an item. Only the user will hear the kaching noise as to not give them away. Ever since playing SS14, having no auditory feedback when purchasing ANYTHING over the uplink has perturbed me quite a bit.

Sound here:

https://github.com/tgstation/tgstation/assets/105099113/bbd518e2-882b-4a12-998d-2b9d00d855bb

## Why It's Good For The Game

Gives actual (auditory) feedback for whenever a purchase is made. Useful if time dilation goes out-of-wack or if you misclicked on your open uplink and dont know why your TC count dropped. (The latter once happened to me. Don't ask.)

## Changelog

:cl:
add: Added a 'kaching' sound to the game, and made uplinks play this sound locally to the user whenever a purchase is made.
/:cl:

### CODE REVIEW

**Comment by hyperjll** [comment]
After LOTS of fuss with Git Bash I.... never actually fixed Git Bash. I had to use the actual website to do things. Fuck whoever made this garbage program.

**Comment by Vect0r2** [comment]
Could you link the sounds in the PR so we can listen to them?

**Comment by Helg2** [comment]
You also have a duplicate sound file.

**Comment by DATA-xPUNGED** [comment]
> > Could you link the sounds in the PR so we can listen to them without having to take a look at the branch?
> 
> How do you do that?

image

**Comment by Sadboysuss** [comment]
would this play the sound only for the player that bought it or for everyone nearby? I hope it's not the latter

**Comment by DATA-xPUNGED** [comment]
> would this play the sound only for the player that bought it or for everyone nearby? I hope it's not the latter

read the PR body

File: kaching.ogg
Comment (MrMelbert) [inline kaching.ogg:1]: You seem to have put a `kaching.ogg` in the root folder in addition to the sound effect folder

Comment (hyperjll) [reply kaching.ogg:1]: > You seem to have put a `kaching.ogg` in the root folder in addition to the sound effect folder

Ack, I just noticed that as well. How the hell do i delete that?

**Comment by hyperjll** [comment]
> Could you link the sounds in the PR so we can listen to them without having to take a look at the branch?

Sound is now in the description.

File: kaching.ogg
Comment (hyperjll) [reply kaching.ogg:1]: Nevermind, figured it out.

**Review by Ghommie — APPROVED** [review]

**Pull request merged**